modules/selenium/src/test/java/org/testcontainers/junit/SpecificImageNameWebDriverContainerTest.java

```diff
@@ -0,0 +1,28 @@
+package org.testcontainers.junit;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testcontainers.containers.BrowserWebDriverContainer;
+
+import java.io.IOException;
+
+/**
+ *
+ */
+public class SpecificImageNameWebDriverContainerTest extends BaseWebDriverContainerTest {
+
+    @Rule
+    public BrowserWebDriverContainer firefox = new BrowserWebDriverContainer("selenium/standalone-firefox-debug:2.53.1-beryllium")
+            .withDesiredCapabilities(DesiredCapabilities.firefox());
+
+    @Test
```

Comment:
you can make the parent class abstract with `abstract protected BrowserWebDriverContainer  getContainer()` method. We were using something like that in Zipkin's tests:
https://github.com/openzipkin/zipkin/blob/master/zipkin-storage/mysql/src/test/java/zipkin/storage/mysql/MySQLTest.java